src/game/GameProcMain.cpp

```diff
@@ -584,9 +596,59 @@ void CGameProcMain::Tick() {
         m_pUIChatDlg->ShowContinueMsg();
         fInterval5 = 0;
     }
-    fTimePrev = fTime;
     // Ÿ�̸� ����� ��ƾ..
     ////////////////////////////////////////////////////////////////////////////////////
+
+    //EXIT MENU
+    static float fTimeIntervalExitWarning = 0;
+    fTimeIntervalExitWarning += fTime - fTimePrev;
+
+    if (m_pExitState == EXIT_STATE_DISALLOW_LEAVE && m_pExitSecondsElapsed <= TIME_PLAYER_CAN_EXIT_GAME_AFTER_ATTACK &&
+        m_pExitType == EXIT_TYPE_NONE && fTimeIntervalExitWarning > 1.0f) {
+        m_pExitSecondsElapsed++;
+
+        if (m_pExitSecondsElapsed == TIME_PLAYER_CAN_EXIT_GAME_AFTER_ATTACK) {
+            m_pExitState = EXIT_STATE_ALLOW_LEAVE;
+            m_pExitSecondsElapsed = 0;
+        }
+        fTimeIntervalExitWarning = 0;
+    }
+
+    if (m_pExitType != EXIT_TYPE_NONE) {
+        if (TIME_PLAYER_CAN_EXIT_GAME_AFTER_ATTACK - m_pExitSecondsElapsed > 0 && fTimeIntervalExitWarning > 1.0f) {
+
+            if (m_pUIChatDlg) {
+
+                char        szBuff[32];
```

Comment:
Zero initialize buffers:
```suggestion

                char        szBuff[32]{};
```

Reply:
Fixed here: https://github.com/ko4life-net/ko/pull/166/commits/1879984b41997ce917e9201664d0d80f534b41dd